### PR TITLE
Add HDInsight Spark and SSH workflows

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**
Added: 

- 1 workflow to submit an application to a HDInsight Spark cluster  (using remote SSH script execution)
- 3 workflows for a SSH service via web browser (start, finish and a parent workflow to start the service from the WE portal).

### CODE REVIEW

File: SSH/METADATA.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		"tags": ["SSH", "Terminal", "Web Browser", "Wetty"],
+		"version": "1.0"
+	},
+	"catalog" : {
+		"bucket" : "service-automation",
+		"objects" : [
+			{
+				"name" : "SSH_Service",
+				"metadata" : {
+					"kind": "Workflow/standard",
```
Comment (ImenBizid) [inline SSH/METADATA.json:16]: The kind of this workflow is workflow/psa no ?

Comment (nebilbenmabrouk) [reply SSH/METADATA.json:16]: No this is a standard workflow. It is the parent workflow that calls PSA workflows, it is executed from the WE portal

File: SSH/resources/catalog/SSH_Service.xml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+      </variables>
+      <genericInformation>
+        <info name="task.icon" value="/automation-dashboard/styles/patterns/img/wf-icons/ssh.png"/>
+        <info name="task.documentation" value="PSA/PSAUserGuide.html"/>
+      </genericInformation>
+      <pre>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+
```
Comment (mbenguig) [inline SSH/resources/catalog/SSH_Service.xml:37]: <pre> section is empty -> can be removed ?

File: SSH/resources/catalog/SSH_Service.xml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+      </genericInformation>
+      <depends>
+        <task ref="wait_for_stop"/>
+      </depends>
+      <pre>
+        <script>
+          <code language="groovy">
+            <![CDATA[
+
+]]>
```
Comment (mbenguig) [inline SSH/resources/catalog/SSH_Service.xml:83]: <pre> section can be removed ?

**Review by mbenguig — APPROVED** [review]
2 very small comments (nice!)

File: SSH/resources/catalog/SSH_Service.xml (hunk 3 above)
Comment (nebilbenmabrouk) [reply SSH/resources/catalog/SSH_Service.xml]: Done for both sections. Thanks @mbenguig

**Pull request merged**